src/app/project/list/list-item-form/list-item-form.component.ts

```diff
@@ -86,51 +114,40 @@ export class ListItemFormComponent implements OnInit {
         }
     }
 
-    submitData() {
+    createChildNode() {
```

Comment:
I will be picky: would it be possible to add in this file as well some method description?

Reply:
I tried my best as I don't fully understand some of the methods either 😅 added in be62277